README.md

```diff
@@ -1,2 +1,237 @@
-# publish
-A CLI tool for publishing a node package to npm
+# publish #
+A command line tool and node module that can be used to simplify the process of publishing a module to NPM. This is particularly useful for creating development releases, e.g. nightly or continuous integration releases.
```

Comment:
We will need to include instructions (and, presumably, some extra packaging) for actually installing this as a command line tool via npm install -g - otherwise it is not clear how it can be used - all the the instructions are presumably from the point of view of "publish.js" being in the current directory which in any realistic situation it will not be